src/components/MewNotification/MewNotification.vue

```diff
@@ -1,192 +1,342 @@
+    <!-- NEW -->
+    <!-- ===================================================================================== -->
+    <div class="notification-container">
+      <v-expansion-panels flat>
+        <v-expansion-panel
+          :class="
+            getBorderClasses(
+              notification.status.value.toLowerCase(),
+              notification.read
+            )
+          "
         >
-          <!--
-  =====================================================================================
-    Displays indicator if notification not read
-  =====================================================================================
-  -->
-          <div
-            :class="[
-              getClasses(notification.status.value.toLowerCase()),
-              'indicator',
-              'ml-2',
-              'd-none',
-              'd-sm-flex',
-            ]"
-            v-if="$vuetify.breakpoint.smAndUp && !notification.read && showIndicator"
-          />
-          <!--
-  =====================================================================================
-   Displays blockie if it is not a swap notification 
-  =====================================================================================
-  -->
-          <mew-blockie
-            class="d-flex ml-2"
-            v-if="!isSwap"
-            width="24px"
-            height="24px"
-            :address="notification.from.value"
-          />
-          <!--
-  =====================================================================================
-   Displays swap icons if it is a swap notification 
-  =====================================================================================
-  -->
-          <div
-            v-else
-            class="d-flex flex-column currency-symbol"
+          <v-expansion-panel-header
+            hide-actions
+            class="px-2 py-0"
+            :color="backgroundColor"
           >
-            <img
-              :src="notification.fromObj.icon ? notification.fromObj.icon : ethTokenPlaceholder"
-              width="24px"
-              height="24px"
-            >
-            <img
-              :src="notification.toObj.icon"
+            <v-container fluid class="px-0">
+              <v-row dense>
+                <v-col class="d-flex align-center" cols="8">
+                  <!-- ===================================================================================== -->
+                  <!-- Displays indicator if notification not read -->
+                  <!-- ===================================================================================== -->
+                  <div
+                    v-if="
+                      $vuetify.breakpoint.smAndUp &&
+                      !notification.read &&
+                      showIndicator
+                    "
+                    :class="[
+                      getClasses(notification.status.value.toLowerCase()),
+                      'indicator',
```

Comment:
this indicator styles isn't showing up anymore

Reply:
fixed by adjusting content width

Reply:
oh i meant its not getting the styles from the `.indicator` class for some reason

Reply:
so its a square instead of a circle

Reply:
fixed the indicator shape